website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Helm Based Operator Quickstart
+title: Quickstart for Helm-based Operators
 linkTitle: Quickstart
-weight: 2
+weight: 100
+description: A simple set of instructions that demonstrates the basics of setting up and running a Helm-based operator.
 ---
 
 This guide walks through an example of building a simple nginx-operator powered by [Helm][helm-official] using tools and libraries provided by the Operator SDK.
 
+**Note**: This guide uses [minikube][minikube-tool] version v0.25.0+ as the
```

Comment:
Is this still true for all quickstarts? Not kind?

Reply:
Let's address in a follow-up. As far as I know nothing has changed about this particular requirement. Although we may not actually have an opinion on the cluster type for the quickstart in which case we can probably just remove this line.